### PR TITLE
docs: fixed grammatical and spelling errors

### PR DESCRIPTION
**Pull request by vyasriday**
##### Checklist

- [x] `npm test` passes
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

docs

##### Description of change

fixed grammatical and spelling errors in intro docs in en directory.

### CODE REVIEW

File: docs/source/en/intro/egg-and-koa.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ title: Egg and Koa
...
 
 - [callback hell](http://callbackhell.com/): Notorious "callback hell"。
 - [release zalgo](https://oren.github.io/#/articles/zalgo/): Asynchronous functions may call callback function response data synchronously which would bring inconsistency.
 
-The community has provided many solutions for the problems, the winner is Promise, it is built into ECMAScript 2015. On the basis of Promise, and with the ability of Generator to switch context, we can write asynchronous code in synchronous way with [co] and other third party libraries. Meanwhile [async function], the official solution has been published in ECMAScript 2017 and landed in Node.js 8.
+The community has provided many solutions for the problems and the winner is Promise. It is built into ECMAScript 2015. On the basis of Promise, and with the ability of Generator to switch context, we can write asynchronous code in synchronous way with [co] and other third-party libraries. Meanwhile [async function], the official solution has been published in ECMAScript 2017 and landed in Node.js v8.
```
Comment (atian25) [inline docs/source/en/intro/egg-and-koa.md:11]: not v8, it means 8.x

Comment (vyasriday) [reply docs/source/en/intro/egg-and-koa.md:11]: Sure will keep the previous changes. Should this go into another commit or in the same commit with amend?

Comment (atian25) [reply docs/source/en/intro/egg-and-koa.md:11]: in the same pr.  

we use `squash and merge`, so don't need to amend, just push commits

Comment (vyasriday) [reply docs/source/en/intro/egg-and-koa.md]: Updated the PR.

**Pull request merged**